WordPress/gutenberg
# [RNMobile][Embed block] Fix bottom sheet link color in dark mode

## Description

Fixes Embed block bottom sheet link color in dark mode

## How has this been tested?
- Switch to dark mode
- Add an embed block
- Press "Add link"
- Check the color of link in the bottom sheet with title `Learn more about embeds`
- Should be the same color as "Add link" button

## Screenshots 

Before | After
-- | -- 
![before](https://user-images.githubusercontent.com/1845482/131546399-e0d7b96f-0947-441d-a7be-fc995441121a.png)|![after](https://user-images.githubusercontent.com/1845482/131546393-f42606f7-3a58-438e-ab8f-5fb5ba687fe5.png)


## Types of changes

Bug fix (non-breaking change which fixes an issue)



## Checklist:
